docroot/profiles/custom/cgov_site/themes/custom/cgov/src/styles/_errorPage.scss

```diff
@@ -1,7 +1,6 @@
 // BEGIN 404 page redesign, NCISEO-280
 .error-page {
   font-size: 1.125em;
-  padding: 0 20px;
```

Comment:
Hrmm. Can we do this not in this file. Changing this file will change DCEG Connect. If it is simple enough you can move it to ncids_trans/front-end/legacy/styles (just follow the convention). Then import it from ncids_global. OR, we could make a new entry point for it, and move this into legacy.